internal/proxy/task_index.go

```diff
@@ -165,8 +152,15 @@ func (cit *createIndexTask) parseFunctionParamsToIndex(indexParamsMap map[string
 		if _, ok := indexParamsMap["bm25_avgdl"]; !ok {
 			indexParamsMap["bm25_avgdl"] = "100"
 		}
+
+		if metricType, ok := indexParamsMap["metric_type"]; !ok {
+			indexParamsMap["metric_type"] = "BM25"
+		} else if metricType != "BM25" {
```

Comment:
what if metric_type is empty "" ?

Reply:
providing empty metric type at index creation time leads to error and will fail the request